src/indexing.jl

```diff
@@ -56,6 +56,10 @@ function getindex_disk(a::AbstractArray, i::Integer)
     return only(outputarray)
 end
 getindex_disk(a::AbstractArray, i...) = getindex_disk!(nothing, a, i...)
+getindex_disk(a::AbstractArray, i::ChunkIndex{<:Any,OneBasedChunks}) =
+    a[eachchunk(a)[i.I]...]
+getindex_disk(a::AbstractArray, i::ChunkIndex{<:Any,OffsetChunks}) =
```

Comment:
Not sure why these new methods are necessary? Or have they simply been moved?

Reply:
I just moved them out of the macro for Revise.jl, along with view methods etc

(Had that situation where you need to reload Julia to get code updates)